src/config/set.js

```diff
@@ -1,35 +1,36 @@
 'use strict'
 
-const promisify = require('promisify-es6')
+const configure = require('../lib/configure')
+const toCamel = require('../lib/object-to-camel')
+
+module.exports = configure(({ ky }) => {
+  return async (key, value, options) => {
+    options = options || {}
 
-module.exports = (send) => {
-  return promisify((key, value, opts, callback) => {
-    if (typeof opts === 'function') {
-      callback = opts
-      opts = {}
-    }
     if (typeof key !== 'string') {
-      return callback(new Error('Invalid key type'))
+      throw new Error('invalid key')
```

Comment:
Why the change in message?  Is this for go compatibility or similar?

Reply:
No, I will revert.